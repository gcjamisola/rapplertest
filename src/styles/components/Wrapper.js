import styled from 'styled-components';
import * as utils from '../utils';

const { media } = utils;

const Wrapper = styled.div`
    margin: 0 auto;
    width: 90%;

    ${media.phablet`
        margin: 0;
        width: 100%;
    `}
`;

export { Wrapper };
