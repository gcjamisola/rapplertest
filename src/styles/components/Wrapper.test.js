import React from 'react';
import styled from 'styled-components';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import {Wrapper} from './Wrapper';

test('it works', () => {
  const tree = renderer.create(<Wrapper />).toJSON()
  expect(tree).toMatchSnapshot()
})
