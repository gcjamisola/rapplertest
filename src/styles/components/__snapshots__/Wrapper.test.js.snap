// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  margin: 0 auto;
  width: 90%;
}

@media (max-width:572px) {
  .c0 {
    margin: 0;
    width: 100%;
  }
}

<div
  className="c0"
/>
`;
